megamek/src/megamek/client/bot/princess/BasicPathRanker.java

```diff
@@ -1288,6 +1293,76 @@ private double calcLavaHazard(boolean endHex, boolean jumpLanding, Entity moving
         return Math.round(hazardValue * psrFactor);
     }
 
+    private double calcHazardousLiquidHazard(Hex hex, boolean endHex, Entity movingUnit, MoveStep step) {
+        logger.trace("Calculating hazardous liquid hazard.");
+        int unitDamageLevel = movingUnit.getDamageLevel();
+        double dmg;
+
+        // Hovers/VTOLs are unaffected _unless_ they end on the hex and are in danger of
+        // losing mobility.
+        if (EntityMovementMode.HOVER == movingUnit.getMovementMode()
+            || EntityMovementMode.WIGE == movingUnit.getMovementMode()) {
+            if (!endHex) {
+                logger.trace("Hovering/VTOL while traversing hazardous liquids (0).");
+                return 0;
+            } else {
+                // Estimate chance of being disabled or immobilized over open lava; this is
+                // fatal!
+                // Calc expected damage as ((current damage level [0 ~ 4]) / 4) *
+                // UNIT_DESTRUCTION_FACTOR
+                dmg = (unitDamageLevel / 4.0) * UNIT_DESTRUCTION_FACTOR;
+                logger.trace("Ending hover/VTOL movement over lava ({}).", dmg);
+                return dmg;
+            }
+        }
+
+        double hazardValue = 0;
+
+        // Expected Damage for Hazardous Liquid
+        // 2/6 chance 0 damage
+        // 1/6 chance 1d6/2 : 1.75 damage
+        // 1/6 chance 1d6 : 3.5 damage
+        // 1/6 chance 1d6 + 2 : 5.5 damage
+        // 1/6 chance 2d6 : 7 damage
+        // Total: 2.958333...
+        // Let's use 3
+
+        // Factor in potential to suffer fatal damage.
+        // Dependent on expected average damage / exposed remaining armor *
+        // UNIT_DESTRUCTION_FACTOR
+        int exposedArmor;
+        dmg = 3.0;
+        dmg *= HazardousLiquidPoolUtil.getHazardousLiquidPoolDamageMultiplierForUnsealed(movingUnit);
+        dmg = (int) (Math.floor(dmg / HazardousLiquidPoolUtil.getHazardousLiquidPoolDamageDivisorForInfantry(movingUnit)));
+
+        // After all that math let's make sure we do at least 1 damage
+        // (.6 repeating when normalized for the HLP doing no damage 1/3 of the time)
+        dmg = Math.max(dmg, 0.666);
```

Comment:
As pedantic as this will be... to represent a repeating number, go ahead and use the fraction form of 2.0/3.0 instead.

Probably wont cause an issue but it would be a case of using 1/3 in the comment and something not exactly the reverse of it in the code.